rasa/core/channels/rasa_chat.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, url):
         self.base_url = url
 
     async def _check_token(self, token):
-        url = "{}/user".format(self.base_url)
+        url = "{}/auth/validate".format(self.base_url)
```

Comment:
should this be `auth/verify`?

Reply:
since I couldn't get the build-in /verify to work, we are still using the validate endpoint I wrote